stdlib.sh

```diff
@@ -603,7 +789,7 @@ function __STDLIB_API_1_std::log() { # {{{
 # This function may be overridden
 #
 function __STDLIB_API_1_die() { # {{{
-	[[ -n "${*:-}" ]] && std_DEBUG=1 __STDLIB_API_1_std::log >&2 "FATAL: " "${*}"
+	[[ -n "${*:-}" ]] && std_DEBUG=1 __STDLIB_API_1_std::log >&2 "[  ${std_COLOUR_START_FATAL}FATAL${std_COLOUR_END}  ]" "${*}"
```

Comment:
It might be neater to re-factor as:
```
if [[ -n "${*:-}" ]]; then
    local fatal="FATAL"
    if [[ -n "${std_COLOUR_END:-}" ]]; then
        fatal="[ ${std_COLOUR_START_FATAL:-}${fatal}${std_COLOUR_END:-} ]"
    else
        fatal="${fatal}: "
    fi
    std_DEBUG=1 __STDLIB_API_1_std::log >&2 "${fatal}" "${*}"
fi
```